packages/typescript-plugin/src/index.ts

```diff
@@ -16,6 +16,7 @@ import { createLogger, getProxy } from "./utils";
 
 type Config = {
   prometheusUrl?: string;
+  quickInfoMode?: "prometheus" | "html-comment";
```

Comment:
👍 on the `vscode` suggestion. It's also what I had originally. Regarding the name change for `quickInfoMode` I also agree however I think the name `explorer` isn't great. Maybe something like `docsOutputFormat`? And then instead of `prometheus` the default value would be `prometheus-links`?